splash/defaults.py

```diff
@@ -6,6 +6,9 @@
 
 MAX_TIMEOUT = 90.0
 
+RESPONSE_SIZE_LIMIT = None
+MAX_RESPONSE_SIZE_LIMIT = None
```

Comment:
This two looks very alike so a bit confusing, please add a comment about what each they stands for? 

BTW why do we need two variables for this limit?

Reply:
If I recall right:

- `RESPONSE_SIZE_LIMIT` is the limit to use as default if the Splash request does not specify one.

- `MAX_RESPONSE_SIZE_LIMIT` is the maximum value that a Splash request may specify. If a higher value is specified by a Splash request, `MAX_RESPONSE_SIZE_LIMIT` is used as limit instead.

It is indeed confusing, I should indeed add a comment.